EIPS/eip-7488.md

```diff
@@ -0,0 +1,51 @@
+---
+eip: 7488
+title: Ground Zero
+description: Implements protocol and governance changes required by the community
+author: ProphetXZ (@ProphetXZ)
+discussions-to: https://ethereum-magicians.org/t/eip-ground-zero/15480
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-06-30
+---
+
+## Abstract
+
+This EIP "Ground Zero" suggests to switch mainnet to permissioned to lower gas prices, transaction costs, and improve privacy. The adjustment requires protcol and governance changes. The EIP idea is braking and stopping the IPFS and Swarm store and broadcasting and instead suggests scalling down ethereum ressources, so that the community may be served in a more **sustainable way** in the future. In addition, to mitigate the currently existing problem with increasingly high gas prices, ethereum would benefit from switching to a permissioned mainnet (after successfully deploying a testnet version). This would greatly lower the number of requests and decrease the load for operating nodes. A further improvement would be to remove legacy APIs, so nodes no longer answer unnecessary public API requests. 
+
+## Motivation
+
+Since its humble beginnings, the ethereum project has been open to the public and seen many contributions. The past years, however, have made it clear that such an open ecosystem design leads to a wide array of problems such as ongoing consensus problems, increasingly high transaction costs, cases of fraud, false ownership claims of assets such as the DAO hack and resulting fork (see [EIP-779](./eip-779.md)), illegal contracts and privacy intrusion, data theft, artifical and malicious content, and many more such things. 
+
+Halting the IPFS and Swarm storage process, which enabled and multiplied above problems, as well as historically drained the ethereum ecosystem and infrastructure of important ressources and instead developing use cases for the original community would be a remedy for almost all of these problems.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+Mainnet (ID: 1) MUST be switched to permissioned. The permissioned ethereum blockchain then either applies proof-of-authority consensus (see [EIP-225](./eip-225.md)), or alternatively, a new consensus mechanism. In part, this may be achieved by halting IPFS, Swarm, and Whisper, after a successfull testnet version has been deployed, for which Goerli or Sepolia may provide blue prints to some degree.
```

Comment:
A few comments here:

- You need to specify when the transition will take effect. It has to be deterministic and computable.
- You're using too many words. Just say something to the effect that that Ethereum will transition to PoA using EIP-225 as its consensus mechanism.
- You need to specify the initial EIP-225 validator set.
- IPFS and co. are unrelated to Ethereum, and due to their distributed nature, it's not possible to "stop" them. That section has to be removed.
- Goerli never used ERC-225